### PR TITLE
Improvements to docs

### PR DESCRIPTION
**Pull request by timmoreton**
## Description

* added a 'choosing a network page'
* added grant playbook
* more consistent terminology and expanded stability explanation
* added RC network disclaimer
* typos 
* correct docs `package.json` license to Apache2

### CODE REVIEW

**Review by asaj — APPROVED** [review]

File: packages/docs/celo-codebase/protocol/stability/README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Stability Mechanism
 
-The Celo protocol includes a stability mechanism which includes the following:
+https://www.youtube.com/watch?v=kYhDUmKuGCY
```
Comment (jmrossy) [inline packages/docs/celo-codebase/protocol/stability/README.md:3]: Was it intentional to embed one link (below) but not this one?

File: packages/docs/overview.md [hunk 2]
```diff
@@ -84,29 +84,31 @@ The Governance contract is set as “owner” for all of the Celo Core Contracts
...
 
-Celo's stability mechanism can be seen as the decentralized version of the one-to-one issuance and redemption mechanisms of most fiat-backed stablecoins. Users create new Celo Dollar by sending 1 US Dollar worth of Celo Gold to the reserve, or burn a Celo Dollar by redeeming them for 1 US Dollar worth of Celo Gold.
+In addition, a back-up reserve of cryptocurrencies is held off-chain.  This off-chain reserve is managed to preserve value and minimize volatility by maintaining a diversified portfolio of cryptocurrencies through algorithmic rebalancing trading and periodically "topping-up" the Celo Gold collateral available to ensure it exceeds the amount required to redeem Celo Dollars in circulation.  The approved cryptocurrencies, distribution ratios, and rebalancing period are all subject to on-chain governance.
```
Comment (jmrossy) [inline packages/docs/overview.md]: Nit: Looks like two sentences in this line have an extra leading space. "This off..." and "The approved..."
Is that a markdown feature?

**Review by jmrossy — APPROVED** [review]
This is great! Two nits below but overall lgtm

File: packages/docs/overview.md (hunk 2 above)
Comment (timmoreton) [reply packages/docs/overview.md]: copy-paste :-/

**Pull request merged**